resources/guidelines/documentation-guidelines/05-methodize-assets.md

```diff
@@ -41,11 +57,47 @@ To create a pictorial illustration, reach out to the Shopware design team.
         storefront-dataHandling-pages_01.svg
     ```
 
-* Use `alt text` and figure captions. This details what an image is trying to convey.
-
 * An introductory sentence should precede most images.
 
-* For UML diagrams use [Mermaid](https://mermaid.live/) to create one and add the diagram code in code block.
+* Store all the media in the [assets directory]( https://github.com/shopware/docs/tree/main/assets). Once it is loaded, copy the reference to the Markdown file. Test images in a local build.
+
+## Diagrams
+
+Consider using diagrams when you need to:
+
+* Show architecture
+* Show complex relationships
+* Define a complex workflow
+* For UML diagrams, use [Mermaid](https://mermaid.live/) to create one and add the diagram code in codeblock.
```

Comment:
Maybe we add a little example here?

Reply:
Do we want to only use it on UML? Gantt is also great for timelines etc.

![image](https://github.com/shopware/docs/assets/8600299/6c2196d7-232f-4534-a648-496793399ddf)

Reply:
@Isengo1989 , but we dont commonly come across scenarios that make use of Gnatt charts. However, let me elaborate this text better